SK.WebApp/yarn.lock

```diff
@@ -1938,6 +1952,14 @@ fd-slicer@~1.1.0:
   dependencies:
     pend "~1.2.0"
 
+fetch@^1.1.0:
+  version "1.1.0"
+  resolved "https://registry.yarnpkg.com/fetch/-/fetch-1.1.0.tgz#0a8279f06be37f9f0ebb567560a30a480da59a2e"
+  integrity sha1-CoJ58Gvjf58Ou1Z1YKMKSA2lmi4=
+  dependencies:
+    biskviit "1.0.1"
+    encoding "0.1.12"
+
```

Comment:
You've removed `fetch` from `package.json` but you didn't run `yarn install`. Please run `yarn install` after you manually change the `dependencies` or `devDependencies` in `package.json`

Reply:
Oh, sorry I forgot about do "yarn" before commit.